static/src/javascripts/projects/common/views/breaking-news.html

```diff
@@ -6,10 +6,10 @@
             <div class="breaking-news__item-headline"><%=headline%></div>
         </div>
         <div class="breaking-news__item-options">
-            <button class="js-breaking-news__item__close button button--tertiary u-faux-block-link__promote"
+            <button class="js-breaking-news__item__close button button--tertiary"
                     aria-label="Dismiss"
                     data-link-name="close button"><%=closeIcon%></button>
         </div>
     </div>
-    <a class="u-faux-block-link__overlay" href="/<%=id%>"></a>
+    <a href="/<%=id%>"></a>
```

Comment:
I believe we need to wrap the button in an `<Object>` here and the `<a>` needs to overlay the contents of this container as it did when it had the `.u-faux-block-link__overlay` class as now it'll have no height/width.

Reply:
@SiAdcock and me have spoken about this already so the comment above is a note to remind us! ☝️

Reply:
shouldn't the link wrap everything, rather than overlay?